core/scss/utilities/mixins/button/_button-primary.scss

```diff
@@ -7,17 +7,14 @@
 //
 // Style guide: Mixins.Button.button-primary
 //
-@mixin button-primary {
-  @include font-smoothing;
-  @include margin(0.5em 0.5em 0.5em null);
-  @include padding(1rem 2rem 1.15rem);
 
+%button-base {
```

Comment:
I like where you are going for this. I vote for moving this to its own file and documenting it like we do mixins. Perhaps we need to create a `/core/scss/utilties/placeholders/` directory?

Reply:
Yeah, I think that makes sense - do you think it's better to add a directory for each component that has placeholders or just all in the root /placeholders?

Reply:
Directories are good. We will likely get to a point where we will want them so let's start with them.

Reply:
@sherakama , I just pushed up a commit. I created `/placeholder` inside `/utilities`. I'm using the same structure as `/variables` by creating a `/components` inside `/placeholders` and moved the button placeholder style there. What do you think? Since we're most likely to have only 1 placeholder for each component, I didn't create a separate `/button` inside `/components`.

TL;DR - we now have
/utilities/placeholders
/utilities/placeholders/components/_button.scss (where new %button-base is)

What do you think?